docs/config_search.md

```diff
@@ -187,9 +184,19 @@ Using the `--run-config-search-<min/max>...` CLI options you have the ability to
 
 Note: That by default quick search runs unbounded and ignores any default values for these settings
 
+## Ensemble Model Search
```

Comment:
Should we explicitly say we do not support BLS? Or is that considered different enough from ensembles?